tasks/tasks.py

```diff
@@ -139,6 +140,26 @@ async def check_concluded_proposals_task(bot: commands.Bot):
             result_message = f"Vote for **{proposal_data['title']}** has concluded:\n\n"
 
             if passed:
+                # Fetch total supply and 25% of it before calling the subprocess
+                total_supply_sum, total_supply_25_percent = fetch_XP_Tokens()
+                logger.info(f"Total supply of all tokens: {total_supply_sum}")
+                logger.info(f"25% of total supply: {total_supply_25_percent}")
+
+                # Modify space settings
+                quorum_value = str(total_supply_25_percent)
+                modify_space_command = [
```

Comment:
imo it would make sense for the whole "modify space" function to be moved to "helpers" or some other file

Reply:
Moved to helpers